package.json

```diff
@@ -47,6 +47,7 @@
     "prettier-eslint": "^9.0.1",
     "prettier-eslint-cli": "^5.0.0",
     "sinon": "^8.0.4",
+    "sinon-chai": "^3.4.0",
```

Comment:
ooh, I don't think we're using this elsewhere, might have to add that to open soon...